apps/docs/src/app/documentation/core-helpers/sections-toolbar/sections-toolbar.component.html

```diff
@@ -9,7 +9,7 @@
             [disabled]="false"
             [ngModel]="search"
             (ngModelChange)="onSearchChange($event)"
-            (addOnButtonClicked)="search = ''"
+            (addOnButtonClicked)="search = ''; onSearchChange('')"
```

Comment:
I guess it should just work without direct assignment to `search` 
```suggestion
            (addOnButtonClicked)="onSearchChange('')"
```